### DISCUSSION TITLE
Do you want to hear about bugs in the 1.4.x API docs?

### DISCUSSION THREAD

**Discussion by adamcameron** [wheels-dev/wheels#1206] [General]
Just checking, before I raise issues.

I think it's completely legit to say "nah mate, the ship has sailed on 1.4.x...", in which case I won't bother :-)

**Comment by bpamiri** [comment]
I don’t see putting out a new version of the 1.4 branch but some of the code you are finding is code that has carried forward to the 2.x branch so it is good to look into and resolve. Basically if the issue is reproducible in the 2.x branch, I’d love to get it fixed. Your analysis and write up are valuable in that regard.

--Peter Amiri
  ***@***.***

**Reply by adamcameron** [reply]
This particular one is fixed in 2.x; I'll always check that first, and I'd def just go ahead and raise the issue if it was wrong there too.

In this case the 1.4.x docs for `findOne` say the `returnAs` defaults to `query`, but it actually defaults to `object`. This bit me y/day.

https://api.cfwheels.org/v1.4.5/findone.html

The docs for 2.x correctly state `object`.

**Comment by bpamiri** [comment]
Let me think through this a little, I may be able to fix the 1.4 docs to reflect the valid default.

**Comment by bpamiri** [comment]
Okay, I was able to correct the docs at least.
